src/main/resources/application.yml

```diff
@@ -18,13 +18,16 @@ okta:
     issuer: ${OKTA_ISSUER:https://dev-18092578.okta.com/oauth2/default}
     audience: ${OKTA_AUDIENCE:api://default}
 
+admin-api-key: ${ADMIN_API_KEY:0a51991c}
+
 client:
   vsac_base_url: https://vsac.nlm.nih.gov/vsac
   valueset_endpoint: /svs/RetrieveMultipleValueSets?id={oid}&profile={profile}&includeDraft={includeDraft}
   default_profile: ${DEFAULT_VSAC_PROFILE:Most Recent Code System Versions in VSAC}
   fhir-terminology-service:
     base-url: ${FHIR_TERMINOLOGY_BASE_URL:https://uat-cts.nlm.nih.gov/fhir}
     manifests-urn: /Library
+    code-system-urn: /res/CodeSystem
```

Comment:
Turns out `/res/` is non-standard, though VSAC supports requests both with and without it. SME's are thinking it could VSAC's way of identifying the URL as part of a response. We are just guessing, though.

Since it's non-standard and the standard pattern is supported, let's remove `res` from the constant.